### ISSUE TITLE
understanding snakecharmer dependency implications

### ISSUE THREAD

**Issue by dpastoor** [nlmixrdevelopment/RxODE#155]
From peeking in the code, it seems that snakecharmer at this point is just a fallback if reticulate does not work/is not set up, hence there is nothing in the package that _only_ works via snakecharmer.

Now that it is truly officially removed (archived) from cran https://cran.r-project.org/web/packages/SnakeCharmR/index.html trying to understand the implications of having that package in our MPN snapshots (curated CRAN-like repos at mpn.metworx.com/snapshots/stable/{{date}})

We want to make sure RxODE/nlmixr is fully supported on metworx, which has reticulate on it, so I think dropping snakecharmer is ok in that case?

**Comment by mattfidler** [comment]
Hi Devin,

I will let you decide and tell you what I have done.

My current advice is to use `SnakeCharmR` on windows and `reticulate` on all other systems.  

Depending on the system, `reticulate` doesn't always detect the windows python installation;  This is a bug still open on their github:
 
https://github.com/rstudio/reticulate/issues/366

In our courses where we used reticulate about 50% of windows users couldn't get nlmixr running whereas with snakeCharmR everyone could use the system.

Since `snakeCharmR` is archived, I'm unsure if CRAN will force us to remove it;  I just raised the question with the CRAN maintainers.

Long term, we will no longer require `snakeCharmR` or `reticulate` but we will be using `symengine` and dropping the python dependency in general. 

I asked if CRAN will be pushing `symengine` out or re-animating `snakeCharmR` since we depend on it.
